src/engine.cpp

```diff
@@ -839,13 +839,16 @@ void Engine_MainLoop()
         Sys_ResetTempMem();
         Engine_PollSDLEvents();
 
-        gl_text_line_p fps = GLText_OutTextXY(screen_info.w - 10.0f, 10.0f, fps_str);
-        if(fps)
-        {
-            fps->x_align    = GLTEXT_ALIGN_RIGHT;
-            fps->y_align    = GLTEXT_ALIGN_BOTTOM;
-            fps->font_id    = FONT_PRIMARY;
-            fps->style_id   = FONTSTYLE_MENU_TITLE;
+        if (renderer.settings.show_fps)
```

Comment:
Thanks for PR!
In OT spaces are used, no tabs. after replacing to spaces will be merged.

Reply:
Nice catch! I didn't expect that - damn Beyond Compare ;}
Fixed!